arches/app/views/user.py

```diff
@@ -159,12 +164,26 @@ def post(self, request):
                 user = form.save()
                 try:
                     admin_info = settings.ADMINS[0][1] if settings.ADMINS else None
+                    email_username = user.username
+                    # Use firstname for email if we have it.
+                    if user.first_name != "" and user.first_name is not None:
```

Comment:
this can be simplified to `if user.first_name:`

Reply:
Amended as per request